src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,15 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': todo.completed,
+  })}
+  >
+    <h2 className="TodoInfo__title">
+      {todo.title}
+    </h2>
+    <UserInfo user={todo.user} />
```

Comment:
```suggestion
    <h2 className="TodoInfo__title">
      {todo.title}
    </h2>

    <UserInfo user={todo.user} />
```
the previous comment is not fixed - add the blank line between sibling elements to increase readability
not critical, but I'll reject PR for now because there are a few more things that might be improved too